pom.xml

```diff
@@ -297,6 +297,7 @@
 								<allow-uri-read>true</allow-uri-read>
 								<toclevels>3</toclevels>
 								<numbered>true</numbered>
+								<baseDir>${project.basedir}</baseDir>
```

Comment:
Look for `{baseDir}` in https://github.com/gregturn/spring-hateoas/blob/enhanced-follow/src/main/asciidoc/index.adoc.